### ISSUE TITLE
Unable to install with yarn2

### ISSUE THREAD

**Issue by wilsonianb** [coilhq/tigerbeetle-node#7]
```
$ yarn add tigerbeetle-node
➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed
➤ YN0000: ┌ Fetch step
➤ YN0013: │ tigerbeetle-node@npm:0.2.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0000: └ Completed
➤ YN0000: ┌ Link step
➤ YN0007: │ tigerbeetle-node@npm:0.2.4 must be built because it never did before or the last one failed
➤ YN0009: │ tigerbeetle-node@npm:0.2.4 couldn't be built successfully (exit code 1, logs can be found here: /tmp/xfs-ebfe6f00/build.log)
➤ YN0009: │ tigerbeetle-node@npm:0.2.4 couldn't be built successfully (exit code 1, logs can be found here: /tmp/xfs-ebfe6f00/build.log)
➤ YN0009: │ tigerbeetle-node@npm:0.2.4 couldn't be built successfully (exit code 1, logs can be found here: /tmp/xfs-baa20dcb/build.log)
➤ YN0009: │ tigerbeetle-node@npm:0.2.4 couldn't be built successfully (exit code 1, logs can be found here: /tmp/xfs-baa20dcb/build.log)
➤ YN0000: └ Completed in 0s 644ms
➤ YN0000: Failed with errors in 0s 700ms
```
```
cat /tmp/xfs-ebfe6f00/build.log
# This file contains the result of Yarn building a package (tigerbeetle-node@npm:0.2.4)
# Script name: preinstall

Internal Error: tigerbeetle-node@workspace:.: This package doesn't seem to be present in your lockfile; try to make an install to update your resolutions
    at J.getCandidates (/home/brandon/test/test-yarn/.yarn/releases/yarn-berry.cjs:2:348376)
    at n.getCandidates (/home/brandon/test/test-yarn/.yarn/releases/yarn-berry.cjs:2:337366)
    at /home/brandon/test/test-yarn/.yarn/releases/yarn-berry.cjs:2:359022
    at Module.w (/home/brandon/test/test-yarn/.yarn/releases/yarn-berry.cjs:2:418493)
    at C (/home/brandon/test/test-yarn/.yarn/releases/yarn-berry.cjs:2:358985)
```

```
cat /tmp/xfs-baa20dcb/build.log
# This file contains the result of Yarn building a package (tigerbeetle-node@npm:0.2.4)
# Script name: postinstall

Internal Error: tigerbeetle-node@workspace:.: This package doesn't seem to be present in your lockfile; try to make an install to update your resolutions
    at J.getCandidates (/home/brandon/test/test-yarn/.yarn/releases/yarn-berry.cjs:2:348376)
    at n.getCandidates (/home/brandon/test/test-yarn/.yarn/releases/yarn-berry.cjs:2:337366)
    at /home/brandon/test/test-yarn/.yarn/releases/yarn-berry.cjs:2:359022
    at Module.w (/home/brandon/test/test-yarn/.yarn/releases/yarn-berry.cjs:2:418493)
    at C (/home/brandon/test/test-yarn/.yarn/releases/yarn-berry.cjs:2:358985)
```
It sounds like yarn hides output from at least `postinstall`, so I think the log files are missing details.
https://blog.typicode.com/husky-git-hooks-autoinstall/
https://github.com/yarnpkg/yarn/issues/5476

I think the current `postinstall` script could be removed since `yarn build` is run in `prepack` and the [README](https://github.com/coilhq/tigerbeetle-node#installation) says to run `yarn` and `yarn build` for local development.

I'm not sure what if anything should change with `preinstall`.

**Comment by jorangreef** [comment]
Thanks @wilsonianb we're looking into this.

**Comment by DonChangfoot** [comment]
Closed by 227c41cacebd32cbdc782cd7343355b23e33a21a
@wilsonianb Thanks for testing this out. v0.2.12 should fix this. Feel free to reopen if you are still running into issues.
